From 6468a0abffe30b2e7a9a245bb4a78fc843d97cd2 Mon Sep 17 00:00:00 2001
From: Arnd Bergmann <arnd@arndb.de>
Date: Sat, 12 Dec 2015 08:53:21 +0100
Subject: [PATCH] reset: hi6220: fix modular build

We need to include <linux/module.h> to build the driver as a loadable
module:

drivers/reset/hisilicon/hi6220_reset.c:108:1: warning: data definition has no type or storage class
 postcore_initcall(hi6220_reset_init);

Signed-off-by: Arnd Bergmann <arnd@arndb.de>
---
 drivers/reset/hisilicon/hi6220_reset.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/reset/hisilicon/hi6220_reset.c b/drivers/reset/hisilicon/hi6220_reset.c
index d17c910..7787a9b 100644
--- a/drivers/reset/hisilicon/hi6220_reset.c
+++ b/drivers/reset/hisilicon/hi6220_reset.c
@@ -12,6 +12,7 @@
 
 #include <linux/io.h>
 #include <linux/init.h>
+#include <linux/module.h>
 #include <linux/bitops.h>
 #include <linux/of.h>
 #include <linux/reset-controller.h>
-- 
1.9.1

